src/CarbonAware.WebApi/src/README.md

```diff
@@ -165,3 +165,20 @@ Controllers are responsible for managing the "Success" responses.  If an error o
 The .Net framework will automatically respond to validation errors with a [ValidationProblemDetails](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.validationproblemdetails?view=aspnetcore-6.0) object.  Using the Exception Filter class enables the WebAPI to consistently respond with the `ValidationProblemDetails` error schema in all error cases and take advantage of error handling automatically provided by the framework.
 
 ![WebAPI Error Handling Flow Chart](docs/images/web-api-error-handling-flow.png)
+
+## Autogenerate WebAPI
+
+Using the following steps, it is possible to get the CarbonAware WebApi OpenAPI specification
+
+1. Make sure the current directory is `<workspace>/carbon-aware-sdk/src/`
```

Comment:
Technically there is no requirement for the root directory to share the same name as the project.  I think anyone trying to run this code will be fine with these instructions either way, but you might consider:

```suggestion
1. Make sure the current directory is `<path to project root>/src/`
```